PWGHF/TableProducer/treeCreatorToXiPi.cxx

```diff
@@ -116,6 +116,10 @@ DECLARE_SOA_COLUMN(NormImpParPiFromCharmBar, normImpParPiFromCharmBar, double);
 DECLARE_SOA_COLUMN(NormDecayLenCharmBar, normDecayLenCharmBar, double);
 DECLARE_SOA_COLUMN(IsPionGlbTrkWoDca, isPionGlbTrkWoDca, bool);
 DECLARE_SOA_COLUMN(PionItsNCls, pionItsNCls, uint8_t);
+DECLARE_SOA_COLUMN(PionTpcCrRows, pionTpcCrRows, int16_t);
```

Comment:
```suggestion
DECLARE_SOA_COLUMN(NTpcRowsPion, nTpcRowsPion, int16_t);
```
These names are very misleading. I don't find out what quantity is stored in the column until I reach the very end of the name. Please start the name with the quantity and then append the attributes. Guidelines are described in the HF documentation.

Reply:
Hello @vkucera, now the names have been updated